principles/fp-019-term-stability.md

```diff
@@ -0,0 +1,144 @@
+```
+   <oboInOwl:consider rdf:resource="http://purl.obolibrary.org/obo/OBI_0001544")>
+``` 
+
+   -  OBO: Use the "consider:" tag with the value set to the CURIE(s) of the relevant term(s)
+```
+   consider: OBI:0001544
+```
+
+Note that some older implementations in OWL used the CURIE method as shown below, but this is not preferred.
+
+```
+   <oboInOwl:consider rdf:datatype="http://www.w3.org/2001/XMLSchema#string">OBI:0001544</oboInOwl:consider>
+```
+<i><b>To obsolete a term, the ontology developer</b></i> MAY:
+
+1) Prepend the string "OBSOLETE. " (this precise string, including the space) to the term definition. NOTE: This MUST be implemented consistently. That is, if applied at all, it has to be applied to every obsoleted term definition.
+2) Indicate the reason(s) for obsoleting:
+  -  OWL: Use the "has obsolescence reason" annotation property from OMO ([IAO:0000231](http://purl.obolibrary.org/obo/IAO_0000231)) with the value set to the IRI of one of the individuals of the "obsolescence reason specification" term [IAO:0000225](http://purl.obolibrary.org/obo/IAO_0000225). See below for example.
+  -  OBO: Use "relationship:" with the CURIE for the annotation property (IAO:0000231) and a CURIE for the specific reason (an individual from the "obsolescence reason specification" term [IAO:0000225](http://purl.obolibrary.org/obo/IAO_0000225)). See below for example. Note that older implementations often used alternative methods (described after the examples). These methods are still valid, but are not preferred.
+
+Examples
+-------
+
+The Ontology for Biomedical Investigations obsolete term "cell lysate MHC competitive binding assay using radioactivity detection" (OBI:0001574) can be deprecated as follows:
+
+OWL: 
+```
+<owl:Class rdf:about="http://purl.obolibrary.org/obo/OBI_0001574">
+    <obo:IAO_0000111>obsolete cell lysate MHC competitive binding assay using radioactivity detection</obo:IAO_0000111>
+    <obo:IAO_0000115>Competitive inhibition of binding assay measuring MHC ligand binding by radioactivity detection using MHC derived from a cell lysate</obo:IAO_0000115>
+    <obo:IAO_0000231 rdf:resource="http://purl.obolibrary.org/obo/IAO_0000227"/>
+    <obo:IAO_0100001 rdf:resource="http://purl.obolibrary.org/obo/OBI_0001544"/>
+    <rdfs:label>obsolete cell lysate MHC competitive binding assay using radioactivity detection</rdfs:label>
+    <owl:deprecated rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">true</owl:deprecated>
+</owl:Class>
+```
+
+OBO: 
+```
+[Term]
+id: OBI:0001574
+name: obsolete cell lysate MHC competitive binding assay using radioactivity detection
+def: "OBSOLETE. Competitive inhibition of binding assay measuring MHC ligand binding by radioactivity detection using MHC derived from a cell lysate." []
+relationship: IAO:0000231 IAO:0000227
+is_obsolete: true
+replaced_by: OBI:0001544
+```
+For OBO format, there are multiple alternatives:
```

Comment:
I don't think this has anything directly to do with OBO format. GO, Uberon, CL, all phenotype ontologies all use the same simple data model e.g. making use of comments because this is what browsers show more prominently, because it's simpler and more consistent. Nothing to do with format. OBO Format can represent annotation assertions at exactly the same level of expressivity as OWL

Reply:
These alternatives are listed for OBO format because they are used 'in the field' so to speak. That is, unlike ontologies in OWL format--which all seem to use the same deprecation methods--OBO formatted ontologies have been using different means. I've confirmed that either way is fine, so we say all this to acknowledge these alternatives.